### PR TITLE
action: drop support of AppImage

### PR DESCRIPTION
**Pull request by xiaoyifang**
AppImage itself is still under development .

Usually it will work in some time and then failed in some other time,  it can not always guarrante the user a stable release with Qt.

Recommend users to build from source or from linux distribution's repository package.  or use flatpak package.

### CODE REVIEW

File: .github/workflows/macos-homebrew-breakpad.yml [hunk 1]
```diff
@@ -203,7 +203,7 @@ jobs:
             Windows users can use either `****-installer.exe` (for installer) or `****.zip` (unzip and run).
             The `goldendict.exe` can be dropped into previous installation's folder (if dependencies aren't changed).
 
-            Linux users can use AppImages or Flatpak.
+            Linux users can use Flatpak or build from source.
             https://flathub.org/zh-Hans/apps/io.github.xiaoyifang.goldendict_ng
```
Comment (shenlebantongying) [inline .github/workflows/macos-homebrew-breakpad.yml:207]:
```suggestion
            https://flathub.org/apps/io.github.xiaoyifang.goldendict_ng
```

File: .github/workflows/macos-homebrew.yml [hunk 2]
```diff
@@ -206,7 +206,7 @@ jobs:
             Windows users can use either `****-installer.exe` (for installer) or `****.zip` (unzip and run).
             The `goldendict.exe` can be dropped into previous installation's folder (if dependencies aren't changed).
 
-            Linux users can use AppImages or Flatpak.
+            Linux users can use Flatpak or build from source.
             https://flathub.org/zh-Hans/apps/io.github.xiaoyifang.goldendict_ng
```
Comment (shenlebantongying) [inline .github/workflows/macos-homebrew.yml:210]:
```suggestion
            https://flathub.org/apps/io.github.xiaoyifang.goldendict_ng
```

File: .github/workflows/ubuntu-6.2.yml [hunk 3]
```diff
@@ -190,7 +185,7 @@ jobs:
             Windows users can use either `****-installer.exe` (for installer) or `****.zip` (unzip and run).
             The `goldendict.exe` can be dropped into previous installation's folder (if dependencies aren't changed).
 
-            Linux users can use AppImages or Flatpak.
+            Linux users can use Flatpak or build from source.
             https://flathub.org/zh-Hans/apps/io.github.xiaoyifang.goldendict_ng
```
Comment (shenlebantongying) [inline .github/workflows/ubuntu-6.2.yml:189]:
```suggestion
            https://flathub.org/apps/io.github.xiaoyifang.goldendict_ng
```

File: .github/workflows/ubuntu.yml [hunk 4]
```diff
@@ -177,7 +172,7 @@ jobs:
             Windows users can use either `****-installer.exe` (for installer) or `****.zip` (unzip and run).
             The `goldendict.exe` can be dropped into previous installation's folder (if dependencies aren't changed).
 
-            Linux users can use AppImages or Flatpak.
+            Linux users can use Flatpak or build from source.
             https://flathub.org/zh-Hans/apps/io.github.xiaoyifang.goldendict_ng
```
Comment (shenlebantongying) [inline .github/workflows/ubuntu.yml:176]:
```suggestion
            https://flathub.org/apps/io.github.xiaoyifang.goldendict_ng
```

File: .github/workflows/windows-6.x.yml [hunk 5]
```diff
@@ -250,7 +250,7 @@ jobs:
             Windows users can use either `****-installer.exe` (for installer) or `****.zip` (unzip and run).
             The `goldendict.exe` can be dropped into previous installation's folder (if dependencies aren't changed).
 
-            Linux users can use AppImages or Flatpak.
+            Linux users can use Flatpak or build from source.
             https://flathub.org/zh-Hans/apps/io.github.xiaoyifang.goldendict_ng
```
Comment (shenlebantongying) [inline .github/workflows/windows-6.x.yml:254]:
```suggestion
            https://flathub.org/apps/io.github.xiaoyifang.goldendict_ng
```

File: .github/workflows/windows.yml [hunk 6]
```diff
@@ -220,7 +220,7 @@ jobs:
             Windows users can use either `****-installer.exe` (for installer) or `****.zip` (unzip and run).
             The `goldendict.exe` can be dropped into previous installation's folder (if dependencies aren't changed).
 
-            Linux users can use AppImages or Flatpak.
+            Linux users can use Flatpak or build from source.
             https://flathub.org/zh-Hans/apps/io.github.xiaoyifang.goldendict_ng
```
Comment (shenlebantongying) [inline .github/workflows/windows.yml:224]:
```suggestion
            https://flathub.org/apps/io.github.xiaoyifang.goldendict_ng

```

File: .github/workflows/windows.yml [hunk 7]
```diff
@@ -220,8 +220,9 @@ jobs:
             Windows users can use either `****-installer.exe` (for installer) or `****.zip` (unzip and run).
             The `goldendict.exe` can be dropped into previous installation's folder (if dependencies aren't changed).
 
-            Linux users can use AppImages or Flatpak.
-            https://flathub.org/zh-Hans/apps/io.github.xiaoyifang.goldendict_ng
+            Linux users can use Flatpak or build from source.
+            https://flathub.org/apps/io.github.xiaoyifang.goldendict_ng
+
```
Comment (shenlebantongying) [inline .github/workflows/windows.yml:225]:
```suggestion

```

**Comment by shenlebantongying** [comment]
The badge looks like this:

image

**Comment by vitaly-zdanevich** [comment]
Maybe after a year you can try AppImage again?

**Comment by Samueru-sama** [comment]
@vitaly-zdanevich https://github.com/pkgforge-dev/GoldenDict-ng-AppImage

**Pull request merged**